### PR TITLE
removing Qualys, adding testssl.sh and some cosmetic changes

### PR DESCRIPTION
**Pull request by xep624**

### CODE REVIEW

File: Document/0x05g-Testing-Network-Communication.md [hunk 1]
```diff
@@ -118,7 +123,7 @@ M3 - Insufficient Transport Layer Protection - https://www.owasp.org/index.php/M
 
 ##### Tools
 
-* Qualys SSL Labs - https://www.ssllabs.com/ssltest/
+* testssl.sh- https://testssl.sh
```
Comment (sushi2k) [inline Document/0x05g-Testing-Network-Communication.md]: Can you please also add o-saft https://www.owasp.org/index.php/O-Saft?

File: Document/0x05g-Testing-Network-Communication.md [hunk 2]
```diff
@@ -284,18 +288,32 @@ Create an SSLContext that uses the TrustManager
...
-#### Black-box Testing
+#### Dynamic Analysis
 
-Black-box Testing can be performed by launching a MITM attack using your prefered Web Proxy to intercept the traffic exchanged between client (mobile application) and the backend server. If the Proxy is unable to intercept the HTTP requests/responses, the SSL pinning is correctly implemented.
+Black-box Testing can be performed by launching a MITM attack using your prefered Web Proxy to intercept [1] the traffic exchanged between client (mobile application) and the backend server. If the Proxy is unable to intercept the HTTP requests/responses, the SSL pinning is correctly implemented.
 
 #### Remediation
 
-The SSL pinning process should be implemented as described on the static analysis section.
+The SSL pinning process should be implemented as described on the static analysis section. You may find helpful in implementing certificate pinning OWASP  guide [2].
```
Comment (sushi2k) [inline Document/0x05g-Testing-Network-Communication.md]: Maybe write the sentence like this:
For further information please check the OWASP certificate pinning guide [2].

File: Document/0x05g-Testing-Network-Communication.md (hunk 1 above)
Comment (xep624) [inline Document/0x05g-Testing-Network-Communication.md]: Done.

File: Document/0x05g-Testing-Network-Communication.md (hunk 2 above)
Comment (xep624) [inline Document/0x05g-Testing-Network-Communication.md]: Done.

**Pull request merged**